build/dependencies.yaml

```diff
@@ -185,7 +185,7 @@ dependencies:
       match: configs\[DistrolessIptables\] = Config{list\.BuildImageRegistry, "distroless-iptables", "v([0-9]+)\.([0-9]+)\.([0-9]+)"}
 
   - name: "registry.k8s.io/go-runner: dependents"
-    version: v2.4.0-go1.24.0-bookworm.0
+    version: v2.4.0-go1.24.1-bookworm.0
```

Comment:
was this image built? to update the image it must have been built in kubernetes/release and promoted to registry.k8s.io

Reply:
Ahhhh not quite sure that process, link me the repo and I'll open a PR

Reply:
github.com/kubernetes/release

though I think we're unlikely to rush this right now given the timing of the release cycle k8s.dev/release